report_print.rb

```diff
@@ -41,7 +41,21 @@ def resources_of_type(report, type)
 end
 
 def color(code, msg, reset=false)
-  colors = {:red => "[31m", :green => "[32m", :yellow => "[33m", :cyan => "[36m", :bold => "[1m", :reset => "[0m", :underline => "[4m"}
+  colors = {
+    :red       => "[31m",
+    :green     => "[32m",
+    :yellow    => "[33m",
+    :cyan      => "[36m",
+    :bold      => "[1m",
+    :underline => "[4m",
+    :reset     => "[0m",
```

Comment:
@ripienaar I put the reset at the end so you can cat report_print.rb and not have everything underlined :)